### DISCUSSION TITLE
[Help] Synctex focuses on the external pdf reader after every build.

### DISCUSSION THREAD

**Discussion by dtetruash** [James-Yu/LaTeX-Workshop#3135] [Q&A]
I'm using LaTeX workshop on Windows with an external PDF reader namely SumatraPDF. I am using the settings provided in the wiki for both the reader and the attached synctex. I like having "build on file change" setting enabled as I see when I make a latex error easier that way. 

The problem comes now that I've enabled synctex that after every build the cursor focuses on SumatraPDF and therefore I stop typing on my document and have to refocus...

How does one prevent this?

**Comment by jlelong** [comment]
SumatraPDF is responsible for taking focus and we cannot do anything from LW side.
You can set [`latex-workshop.synctex.afterBuild.enabled `](https://github.com/James-Yu/LaTeX-Workshop/wiki/View#latex-workshopsynctexafterBuildenabled)  to `false` in order not to call synctex after build. You will still be able to call it manually.
